### ISSUE TITLE
[FEAT]: swipe navigation/controls

### ISSUE THREAD

**Issue by digitalbuddha** [AndroidDev-social/MastodonCompose#36]
### Description

I'd love for us to think about swipe/multi-touch controls rather than lots of buttons. For example things like swipe left to boost or right to reply to a post. Ideally I can keep my thumb in  middle of the screen and still perform most basic functions.

**Comment by crocsandcoffee** [comment]
This was a great shout out. The noisy buttons/ctas on other clients and Twitter always get me..

We could do a cool in-app setting to let the user customize their experience (swipe and gestures versus buttons), or make that decision out right and go with less buttons.

**Comment by digitalbuddha** [comment]
Yeah agreed it can be additive/super user. Similar can go as far as shake/flip/gyroscope based controls :-D

**Comment by evant** [comment]
Just make sure it's accessible! Especially when choosing defaults here.
